examples/alerts/alerts.md

```diff
@@ -275,7 +275,7 @@ rules:
     message: Thanos Sidecar {{$labels.job}} {{$labels.pod}} is unhealthy for {{ $value
       }} seconds.
   expr: |
-    count(time() - max(thanos_sidecar_last_heartbeat_success_time_seconds{job=~"thanos-sidecar.*"}) by (job, pod) >= 300) > 0
+    time() - max(thanos_sidecar_last_heartbeat_success_time_seconds{job=~"thanos-sidecar.*"}) by (job) >= 600
```

Comment:
How about keeping `pod` aggregation? It'd make it easier to pin point the source.

Reply:
@kakkoyun sure I can do that.

Reply:
@kakkoyun because we are using the max() operator I am not sure how we can use the pod aggregation. Right now the expression finds the pod with the max latency. If we use `by (job,pod)` then the `max()` has no effect as pod+job combinations are unique anyway. Do I miss something?